docs/source/index.rst

```diff
@@ -1,10 +1,118 @@
+
+Radiative forcing
+^^^^^^^^^^^^^^^^^
+* Thermal relaxation: Held-Suarez, variations for exoplanets, and a 'top--down' scheme that provides a self-consistent determination of the tropopause 
+* Grey radiation, with or without water vapour feedback
+* Simple, two IR band (with an IR window) scheme, with a simple solar scheme
+* Comprehensive, multi-band schemes using either SOCRATES or RRTM
+
+Convection and boundary-layer schemes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* A very flexible moist and dry convective adjustment scheme, suitable for both Earth and other planetary atmospheres
+* The Betts-Miller scheme (a well-known and much used convective relaxation scheme)
+* A mass flux scheme (relaxed Arakawa-Schubert)
+* A simple boundary-layer scheme
+
+Clouds
+^^^^^^
+* A simple cloud scheme based on relative humidity and inversion strength. 
+
+Surface properties
+^^^^^^^^^^^^^^^^^^
+* Specified sea-surface temperatures
+* A slab ocean with a prognostic SST
+* A swamp land surface with an evaporative resistance
+* A simple bucket model 
+* A simple vegetation model
+* Realistic Earth continental outlines, or user-configurable idealized continental configurations 
+
+Test cases
+^^^^^^^^^^
+Various out of the box test cases are available that should run with no additional configuration. These include:
+
+* The Held-Suarez test case
+* An aquaplanet with a slab ocean and grey radiation
+* A MiMA-like test case using RRTM
+* An axi-symmetric model
+* A giant planet (Jupiter-like)
+* Cases with realistic, Earth-like continents
+
+Planets
+^^^^^^^
+Earth is, naturally, the planet to which Isca has most often been applied. However, models of varying degrees of sophistication of Mars, Jupiter, Titan, Trappist 1-e, and a generic tidally-locked exoplanet have or are being constructed. 
+
+.. figure:: index_fig_trappist.png
+   :scale: 75
+
+   Figure 2. Simulations for the Trappist-1 Habitable Atmosphere Intercomparison (THAI) project. Trappist 1-E is a tidally-locked rocky exoplanet and is a promising candidate for habitable surface conditions. Figure produced by Dr Matthew Henry.
+
+The Future
+^^^^^^^^^^
+Isca, and this documentation, is constantly under development. A simple cloud scheme has recently been incorporated into Isca, and models of other planetary bodies are continually being developed. The next major model addition will be a fully interactive ocean as an integrated component of Isca and therefore configurable with the Python interface and flexible in its nature and use.  We invite contributions from users to add to its capabilities and improve its documentation. 
+
+.. figure:: index_fig_qbo.png
+   :scale: 50
+
+   Figure 3. The Quasi-Biennial Oscillation (QBO), an amazing wave driven periodic reversal of the stratospheric tropical winds on Earth. This was produced in Isca using the convective gravity wave parameterisation modified by Prof. Chaim Garkfinel. Figure produced by Ross Castle.
+
+References
```

Comment:
All these references should be moved to the Reference page (with citations keys as appropriate)